com.wps.WPS.appdata.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
You may provide multiple <component> entries for the different launchable entry points that WPN office exports (ie, one per .desktop file).